src/Applications/GEOSdas_App/fvsetup

```diff
@@ -3353,21 +3381,38 @@ sub set_newradbc {
 sub set_ldasANA {
 
     # initialize Land DAS processing flag
-    #------------------------------------
-    $ldasANA = 0;
+    #------------------------------------ 
+    my($dflt, $ans1, $ans2, $ans3, $ans4);
+    $ldas_ana = 0;
+    $ldasfdbk = 0;
+    $ldas_flag = 0;
+    $dflt = "n";
+    $ldashome = "NULL" ;
+    $ldashome4ens = "NULL" ;
+
+    print "\n-------------\n";
+    print   "LDAS Analysis\n";
+    print   "-------------\n\n";
+
+    $ans1 = query("  Land DAS Analysis (y/n)?", $dflt) ;
+    $ldas_ana = 1 if yes($ans1);
+
+   if ($ldas_ana == 1) {
+      $ans2 = query("  Enable LDAS feedback to model y/n ? ", $dflt);
+      $ldasfdbk = 1 if yes($ans2);
+
+      if ($ldasfdbk ==1 ) {
+          $ldas_flag = 1;
+      }
 
-    # query for processing flag, if LDAS executable found
-    #----------------------------------------------------
-    if (-e "$fvbin/LDASsa_assim_seq.x") {
+      $ans3 = query("LDAS HOME = $fvhome, full path?  ");
+      $ldashome = $ans3 ;
 
-        print "\n-------------\n";
-        print   "LDAS Analysis\n";
-        print   "-------------\n\n";
+      $ans4 = query("LDAS HOME for land ensemble = $fvhome/run/atmens, full path?  ");
+      $ldashome4ens = $ans4 ;
```

Comment:
@rtodling: Here, is it really ok to suggest to the user that $fvhome could be the home directory for the LDAS?  If the user accepts $fvsetup as the LDAS HOME, they probably end up with a mess in $fvhome.  (I doubt that the ADAS and LDAS directories merge cleanly within the same directory tree.)
Rather than suggesting $fvhome, maybe we can construct a simple string that avoids the conflict, something concatenated like $fvhome+'_LDAS'   [in pseudo-code].

Reply:
I'm sorry what's the issue? All I see changing here a print statement. This is just a query for the user ... the user change/should answer whatever needed ... I just can't default to something that does exist!!

Reply:
I fully agree that we do not want to use a variable that did not exist.  But neither do we want to suggest to the user something that will potentially result in chaos.  A fix is in new PR #122.
cc: @saraqzhang